fork-runner/src/main/java/com/shazam/fork/PermissionUtils.java

```diff
@@ -0,0 +1,48 @@
+package com.shazam.fork;
+
+import com.android.ddmlib.*;
+import com.shazam.fork.model.Permission;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.util.List;
+
+import static java.lang.String.format;
+
+public class PermissionUtils {
+
+    private static final NullOutputReceiver NO_OP_RECEIVER = new NullOutputReceiver();
+
+    public static void revokePermissions(@Nonnull String applicationPackage,
```

Comment:
Do these need to be static methods? Why not inject the device and use this as a collaborator

Reply:
👍